src/MICore/Debugger.cs

```diff
@@ -379,8 +379,11 @@ public void Init(ITransport transport, LaunchOptions options)
             FlushBreakStateData();
 
             _transport.Init(this, options, Logger);
+        }
```

Comment:
Actually I'd like to get rid of the launch option so that there isn't any issue with conflicts. is there some way to indicate that the use of the attribute is deprecated?